apps/hashdotdev/src/components/mdx-page-content.tsx

```diff
@@ -1,8 +1,9 @@
-// @todo update from blockprotocol
```

Comment:
@CiaranMn @nathggns Any idea what this comment was originally doing here, and whether we should persist it? I don't know if it's appropriate that we remove it (which implies it could've done with more detail).

Reply:
It comes all the way from my PR (recreated/merged by Maggie) which introduced the blog – https://github.com/hashintel/hash/pull/508/files#diff-0e23b4ea81c2ad7fa5a58d05355f073a4852616a21cd96f8fcbd69a25ab357bd 

I think what it was referring to was to update the MDX system to match the one used in blockprotocol, but I don't recall why. I don't think it's particularly worth keeping.